Makefile

```diff
@@ -18,17 +18,9 @@ help:  ## Print the help documentation
 install_tools:  ## Install tools needed for project.
 	scripts/install_tools
 
-.PHONY: ensure_venv
-ensure_venv:  ## Ensure that the virtualenv is activated.
-ifndef VIRTUAL_ENV
-	@echo "Virtual env not defined. If you are using nix, make sure it's not disabled. If you aren't using nix, run 'make setup'."
-	false
-endif
-
 .PHONY: install_python_deps
-install_python_deps: ensure_venv ## Install all python dependencies/requirements
-	pip install -r requirements.txt
-	pip install -r requirements-dev.txt
+install_python_deps: ## Install all python dependencies/requirements
```

Comment:
Can we actually just get rid of this one? Direnv is handling pipenv no? If so, we could also get rid of the `setup` target since at that point it would just be an alias for `ensure_pre_commit`

Reply:
Since we don't have much experience with pipenv yet, can we leave it in to see if we ever need it? We can always remove it in the future